core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/_entryFormPage.jelly

```diff
@@ -44,15 +44,14 @@ THE SOFTWARE.
       The wrapper for the data provided by the user and the error message(s) if any. Is null on first form display.
     </st:attribute>
   </st:documentation>
-  <l:layout title="${title}">
-    <st:adjunct includes="hudson.security.HudsonPrivateSecurityRealm._entryFormPage.resources" />
-    <l:hasPermission permission="${app.READ}" it="${host}">
-      <st:include page="sidepanel.jelly" it="${host}" />
-    </l:hasPermission>
+  <l:layout title="${title}" type="one-column">
+    <l:breadcrumb title="${title}"/>
     <l:main-panel>
-      <form action="${rootURL}/securityRealm/${action}" method="post">
+      <form action="${rootURL}/securityRealm/${action}" method="post" class="jenkins-form jenkins-form-item--small">
```

Comment:
Looks a bit off for me on both big screens and mobile

27" monitor
![image](https://user-images.githubusercontent.com/21194782/193473825-a88274aa-903b-41a0-8602-28d515a7f6ff.png)

iphone XR from chrome devtools:
![image](https://user-images.githubusercontent.com/21194782/193473810-2bd3c3d7-e8bc-4a11-990b-b32078819f76.png)

would centring it help maybe?